ts/torch_handler/distributed/base_neuronx_continuous_batching_handler.py

```diff
@@ -385,26 +379,19 @@ def __call__(self, res):
         )
 
     def _set_class(self, ctx):
-        model_class_name = ctx.model_yaml_config.get("handler", {}).get(
-            "model_class_name", None
-        )
+        handler_config = ctx.model_yaml_config.get("handler", {})
+        model_class_name = handler_config.get("model_class_name", None)
 
         assert model_class_name is not None
```

Comment:
A more elaborate error message in case this fails would be valuable for the user.